content/id/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,140 @@
+      * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` nilai bawaannya adalah /24 
+        untuk IPv4 dan /64 untuk IPv6
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< caution >}}
+Jika kamu menentukan blok alamat IPv6 yang lebih besar dari /24 melalui 
+`--cluster-cidr` pada baris perintah, maka penetapan tersebut akan gagal.
+{{< /caution >}}
+
+## Service
+
+Jika klaster kamu mengaktifkan jaringan _dual-stack_ IPv4/IPv6, maka kamu dapat 
+membuat {{<glossary_tooltip text ="Service" term_id ="service">}} dengan 
+alamat IPv4 atau IPv6. Kamu dapat memilih keluarga alamat untuk clusterIP  
+Service kamu dengan mengatur bagian, `.spec.ipFamily`, pada Service tersebut.
+Kamu hanya dapat mengatur bagian ini saat membuat Service baru. Mengatur bagian 
+`.spec.ipFamily` bersifat opsional dan hanya boleh digunakan jika kamu berencana
+untuk mengaktifkan {{<glossary_tooltip text ="Service "term_id ="service">}} 
+dan {{<glossary_tooltip text ="Ingress "term_id ="ingress">}} IPv4 dan IPv6
+pada klaster kamu. Konfigurasi bagian ini bukanlah syarat untuk lalu lintas 
+[_egress_] (#lalu-lintas-egress).
+
+{{< note >}}
+Keluarga alamat bawaan untuk klaster kamu adalah keluarga alamat dari rentang 
+clusterIP Service pertama yang dikonfigurasi melalui opsi 
+`--service-cluster-ip-range` pada kube-controller-manager.
+{{< /note >}}
+
+Kamu dapat mengatur `.spec.ipFamily` menjadi salah satu dari:
+
+   * `IPv4`: Dimana server API akan mengalokasikan IP dari `service-cluster-ip-range` yaitu `ipv4`
+   * `IPv6`: Dimana server API akan mengalokasikan IP dari `service-cluster-ip-range` yaitu `ipv6`
+
+Spesifikasi Service berikut ini tidak memasukkan bagian `ipFamily`.
+Kubernetes akan mengalokasikan alamat IP (atau yang dikenal juga sebagai 
+"_cluster IP_") dari `service-cluster-ip-range` yang dikonfigurasi pertama kali
+untuk Service ini.
+
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
+
+Spesifikasi Service berikut memasukkan bagian `ipFamily`. Sehingga Kubernetes
+akan mengalokasikan alamat IPv6 (atau yang dikenal juga sebagai "_cluster IP_") 
+dari `service-cluster-ip-range` yang dikonfigurasi untuk Service ini.
+
+{{< codenew file="service/networking/dual-stack-ipv6-svc.yaml" >}}
+
+Sebagai perbandingan, spesifikasi Service berikut ini akan dialokasikan sebuah alamat
+IPv4 (atau yang dikenal juga sebagai "_cluster IP_") dari `service-cluster-ip-range` 
+yang dikonfigurasi untuk Service ini.
+
+{{< codenew file="service/networking/dual-stack-ipv4-svc.yaml" >}}
+
+### Tipe _LoadBalancer_
+
+Penyedia layanan _cloud_ yang mendukung IPv6 untuk pengaturan beban eksternal,
+Mengatur bagian `type` memjadi` LoadBalancer` sebagai tambahan terhadap mengatur bagian 
```

Comment:
```suggestion
Mengatur bagian `type` menjadi` LoadBalancer` sebagai tambahan terhadap mengatur bagian 
```
nit

Reply:
Done! hopefully.. this is the last fixed ;)